### PR TITLE
Release: updated README and RELEASE notes for Apollo 6.0

### PR DESCRIPTION
**Pull request by jinghaomiao**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -311,6 +162,8 @@ Congratulations! You have successfully built out Apollo without Hardware. If you
 
 * [Apollo 5.5 QuickStart Guide](docs/quickstart/apollo_5_5_quick_start.md)
 
+* [Apollo 6.0 QuickStart Guide](docs/quickstart/apollo_6_0_quick_start.md)
+
```
Comment (storypku) [inline README.md:166]: Generally speaking, the more recent ones should be put upfront.

File: RELEASE.md [hunk 2]
```diff
@@ -1,7 +1,28 @@
+# Release 6.0
+
+Apollo 6.0 incorporates new deep learning models to enhance the capabilities for certain Apollo modules. This version works seamlessly with new addtions of data pipeline services to better serve Apollo developers. Apollo 6.0 is also the first version to integrate certain features as a demonstration of our continuous exploration and experimentation efforts towards driverless technology.
+
```
Comment (storypku) [inline RELEASE.md:4]: Typo of `additions`.

File: RELEASE.md [hunk 3]
```diff
@@ -1,7 +1,28 @@
...
+    * Control profiling service
+    * Vehicle dynamic model training service
+    * Open space planner profiling service
+    * Complete control parameter auto-tune service
+* Driverless Research
+    * Remote control interface with DreamView integration
+    * Audio bssed emergency vehicle detection system
+
+**[Note]** All models and methodogies included in Apollo 6.0 are for research purposes only. Productized and commercial uses of these models are **NOT** encouraged, and it is at your own risk. Please be cautious to try Apollo 6.0 with enough safety protection mechanmism and provide your feedback so that we can continuously improve the models.
+
```
Comment (storypku) [inline RELEASE.md:22]: `and provide your feedback ...` -> and any feedback is welcome for us to continuously improve the models

File: docs/howto/how_to_debug.md [hunk 4]
```diff
@@ -71,7 +71,7 @@ Start the module to be debugged, either by using the command line or by using
 the Dreamview interface. The following is an example of debugging the
 **Planning** module from the Dreamview interface.
 
-- Open Chrome browser, enter the URL: http://localhost:8888/
+- Open Chrome browser, enter the URL: <http://localhost:8888/>
 - Open the Dreamview interface
```
Comment (storypku) [inline docs/howto/how_to_debug.md:75]: We don't say enter the url, we say `Open http://xxxx in your favorite web browser (e.g. Chrome).`

File: docs/howto/how_to_debug.md [hunk 5]
```diff
@@ -171,7 +171,7 @@ ifconfig
 
 Assuming that the IP address of the industrial computer LAN is: `192.168.3.137`,
 open your Chrome or Firefox browser, enter the following URL:
-http://192.168.3.137:8888/, start the module (`Planning`) to be debugged as
+<http://192.168.3.137:8888/>, start the module (`Planning`) to be debugged as
 shown in
 [Start the module that needs debugging](#Start-the-module-that-needs-to-be-debugged)
 section.
```
Comment (storypku) [inline docs/howto/how_to_debug.md:177]: Start the module to debug.

File: docs/quickstart/apollo_3_5_hardware_system_installation_guide.md [hunk 6]
```diff
@@ -41,118 +40,117 @@ The **Apollo Project** is an initiative that provides an open, complete, and rel
...
+    * Apollo Extension Unit (AXU)
 
-    - **Software**: Refer to the [Software Installation Guide](https://github.com/ApolloAuto/apollo/blob/master/docs/specs/Software_and_Kernel_Installation_guide.md) for information on the following:
-      - Ubuntu Linux
-      - Apollo Linux Kernel
-      - NVIDIA GPU Driver
+  * **Software**: Refer to the [Software Installation Guide](https://github.com/ApolloAuto/apollo/blob/master/docs/specs/Software_and_Kernel_Installation_guide.md) for information on the following:
+    * Ubuntu Linux
+    * Apollo Linux Kernel
+    * NVIDIA GPU Driver
```
Comment (storypku) [inline docs/quickstart/apollo_3_5_hardware_system_installation_guide.md:62]: This document was deprecated and removed.
The new document was docs/quickstart/apollo_software_installation_guide.md

File: docs/quickstart/apollo_5_0_quick_start.md [hunk 7]
```diff
@@ -2,7 +2,7 @@
 
 The following guide serves as a user manual for launching the Apollo upgraded software and hardware stack on vehicle.
 
-The the latest Apollo upgrade, Quick Start Guide focuses on the new features. For general Apollo concepts, please refer to
+The latest Apollo upgrade, Quick Start Guide focuses on the new features. For general Apollo concepts, please refer to
 [Apollo 3.5 Quick Start](https://github.com/ApolloAuto/apollo/blob/master/docs/quickstart/apollo_3_5_quick_start.md)
```
Comment (storypku) [inline docs/quickstart/apollo_5_0_quick_start.md:6]: 「The latest Apollo upgrade, Quick Start Guide」what does this mean ? `The latest upgrade of Apollo Quick Start Guide` ?

**Review by storypku — DISMISSED** [review]
Please use `./scripts/mdfmt.sh path/to/doc_a.md  path/to/doc_b.md` to format all the markdown files.

And, git rebase your commits. Otherwise, the whole PR will be squashed into one as many comments were titled  `update README.md`

File: docs/quickstart/apollo_6_0_quick_start.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
+# Apollo 6.0 Quick Start Guide
+
+The following guide serves as a user manual for launching the Apollo upgraded software and hardware stack on vehicle, which
+is similar to Apollo 5.5.
+
+The latest Apollo upgrade, Quick Start Guide focuses on the new features, like software and third party dependencies update. For general Apollo concepts, please refer to
```
Comment (storypku) [inline docs/quickstart/apollo_6_0_quick_start.md:6]: And this one.

File: docs/quickstart/apollo_6_0_quick_start.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+
+- [Calibration Guide](#calibration-guide)
+- [Hardware and Software Installation](#hardware-and-software-installation)
+- [Dreamview Usage Table](#dreamview-usage-table)
+- [Onboard Test](#onboard-test)
+
+## Calibration Guide
+
+Apollo currently offers a robust calibration service to support your calibration requirements from LiDARs to IMU to Cameras. This service is currently being offered to select partners only as part of our Data Pipeline service offering. If you would like to learn more about the calibration service, please reach out to us via email: **apollopartner@baidu.com**
+
```
Comment (storypku) [inline docs/quickstart/apollo_6_0_quick_start.md:19]: 1) Two `currently` seems repetitive. The first one can be removed.
2) select partners -> selected partners

File: docs/quickstart/apollo_6_0_quick_start.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+1. Plug-in an external hard-drive to any available USB port in the host machine
+
+2. Turn on the vehicle, and then the host machine
+
+3. Launch the Dev Docker Container
+
+4. Launch DreamView
+
+   Note\: Use Chrome or other browser to access Dreamview web service in your host
+   machine browser with URL <http://localhost:8888>
```
Comment (storypku) [inline docs/quickstart/apollo_6_0_quick_start.md:44]: Why need to escape `:`.

File: README.md [hunk 11]
```diff
@@ -21,138 +21,60 @@ For business and partnership, please visit [our website](http://apollo.auto).
...
+3. [Individual Versions](#individual-versions)
+4. [Architecture](#architecture)
+5. [Installation](#installation)
+6. [Quick Starts](#quick-starts)
+7. [Documents](#documents)
 
-## Getting Started
+## Introduction
 
 Apollo is loaded with new modules and features but needs to be calibrated and configured perfectly before you take it for a spin. Please review the prerequisites and installation steps in detail to ensure that you are well equipped to build and launch Apollo. You could also check out Apollo's architecture overview for a greater understanding of Apollo's core technology and platform.
```
[the remark this answers is not in the document]
Comment (xiaoxq) [reply README.md:35]: Maybe not?

File: README.md [hunk 12]
```diff
@@ -21,138 +21,60 @@ For business and partnership, please visit [our website](http://apollo.auto).
...
- - Please note, it is recommended that you install the versions of Apollo in the following order:
- **1.0 -> whichever version you would like to test out**.
- The reason behind this recommendation is that you need to confirm whether individual hardware components
- and modules are functioning correctly and clear various version test cases,
- before progressing to a higher, more capable version for your safety and the safety of those around you.
+* nVidia container toolkit ([Official doc](https://github.com/NVIDIA/nvidia-docker))
 
-#### Individual Version Requirements:
+**Please note**, it is recommended that you install the versions of Apollo in the following order: **1.0 -> whichever version you would like to test out**. The reason behind this recommendation is that you need to confirm whether individual hardware components and modules are functioning correctly and clear various version test cases, before progressing to a higher, more capable version for your safety and the safety of those around you.
+
```
[the remark this answers is not in the document]
Comment (jinghaomiao) [reply README.md:53]: It's the puncation problem. Will fix it.

File: modules/planning/README.md [hunk 13]
```diff
@@ -16,9 +16,9 @@ use cases as they were all treated as a single driving scenario.
...
 
-**Note:** The current development of E2E mode is in early stage where it is fined-tuned more for the dynamic nudge scenario rather than comprehensive ones. So suboptimality is expected when running other scenarios in simulation. E2E mode and Hybrid mode are not tested on real vehicle yet. The work on data-driven planning is still in progress and welcomes any collaboration by any means including algorithms, models and data.
+**Note:** The current development of E2E mode is in an early stage where the model is trained for the dynamic nudge scenario as a research demonstration purpose. The capability of the model is limited, and suboptimality is expected when it is tested with a wider selection of  scenarios. E2E mode and Hybrid mode are not tested on real roads yet, but rather serve as a baseline to promote and encourage extensive research on learning based planning.  All developers are welcomes to collaborate with us by any means including algorithms, models and data.
```
Comment (storypku) [inline modules/planning/README.md:22]: welcomed

File: README.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (storypku) [reply README.md:35]: > Maybe not?

For example, D-Kit is a hardware platform different from general ADCs.

File: README.md [hunk 14]
```diff
@@ -279,21 +128,23 @@ The car has complete 360-degree visibility, along with upgraded perception deep
...
 
 ## Installation
 
-* [Fork and then Clone Apollo's GitHub code](https://github.com/ApolloAuto/apollo)
+* [Hardware installation guide](docs/quickstart/apollo_3_5_hardware_system_installation_guide.md)
+
+* [Fork and then clone Apollo's GitHub code](https://github.com/ApolloAuto/apollo)
 
-* [Build and Release using Docker](docs/howto/how_to_build_and_release.md) - This step is required
+* [Build with Docker](docs/howto/how_to_build_and_possible_error.md) - **This step is required**
```
[the remark this answers is not in the document]
Comment (jinghaomiao) [reply README.md:142]: Will not fix it now and double check related docs before making new changes.

File: docs/howto/how_to_debug.md (hunk 5 above)
Comment (storypku) [reply docs/howto/how_to_debug.md:174]: This doesn't match: `[Start the module that needs debugging](#Start-the-module-that-needs-to-be-debugged)`.

File: modules/planning/README.md [hunk 15]
```diff
@@ -16,9 +16,9 @@ use cases as they were all treated as a single driving scenario.
 Apollo 5.5, which focuses on curb-to-curb autonomous driving on urban roads,
 introduced 2 new planning scenarios.
 
-Apollo 6.0, which focuses on data-driven development of autonomous driving, introduced two new planning modes: E2E mode and Hybrid mode, by which we demonstrate the new capability of dynamically nudginng a moving obstacle. These two modes also define a series of APIs where developers can generate thier own training data and bring in their own models.
+Apollo 6.0 extended the technology to incorporate data-driven mothedologies to tackle trajectory planning problems with learning-based models, and introduced two new planning modes: E2E mode and Hybrid mode, by which the new capability of dynamically nudginng moving obstacles are demonstrated. In these two modes, a series of APIs is also defined  where developers can generate their own training data and integrate their own models.
```
Comment (storypku) [inline modules/planning/README.md:20]: is also defined -> are also defined

**Review by xiaoxq — APPROVED** [review]

**Pull request merged**